front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -208,6 +221,51 @@ export const usePathfinding = (
     }
   }, [origin?.id, destination?.id, rollingStock]);
 
+  const handleInvalidPathItems = (
+    pathfindingResult: PathfindingResult,
+    pathSteps: (PathStep | null)[],
+    dispatch: AppDispatch,
+    pathfindingDispatch: React.Dispatch<PathfindingAction>,
+    t: TFunction
+  ) => {
+    if (
+      pathfindingResult.status === 'failure' &&
+      pathfindingResult.failed_status === 'pathfinding_input_error' &&
+      pathfindingResult.error_type === 'invalid_path_items'
+    ) {
+      //TODO: we currently only handle invalid pathSteps with trigram. We will have to do it for trackOffset, opId and uic too.
+      const invalidPathItems = pathfindingResult.items
```

Comment:
You can rename this props
```suggestion
      const invalidTrigrams = pathfindingResult.items
```

Reply:
done